CentralComputing/Pod.h

```diff
@@ -29,6 +28,7 @@ class Pod {
 
   private:
     void logic_loop();
+    bool switchVal;
```

Comment:
Moved from a global to a class variable. Since it is only used within the Pod context, it doesn't need to be global